api/manifest.go

```diff
@@ -42,6 +42,8 @@ func gitHubReleaseURL(tag string) string {
 	return fmt.Sprintf(`https://api.github.com/repos/w3c/web-platform-tests/releases/tags/%s`, tag)
 }
 
+const gitHubLatestReleaseURL = `https://api.github.com/repos/w3c/web-platform-tests/releases/latest`
```

Comment:
Ah, excellent, I thought we'd have to figure out what the master sha was and just hope there was a manifest. That wouldn't have been as reliable as this :)

Are the backticks here meaningful, or just copypasta?

Reply:
Some editors use it as a convention when defining consts, but it's only ever different for some chars. No effect in this case.